app/models/orchestration_template/orchestration_parameter_fqname.rb

```diff
@@ -0,0 +1,5 @@
+class OrchestrationTemplate
+  class OrchestrationParameterFqname < OrchestrationParameterConstraint
```

Comment:
`OrchestrationParameterAllowedDynamic` is a better name. There are other dynamic types of controls that we may add in the future.